tests/common/config_reload.py

```diff
@@ -217,6 +217,6 @@ def _config_reload_cmd_wrapper(cmd, executable):
     if wait_for_bgp:
         bgp_neighbors = sonic_host.get_bgp_neighbors_per_asic(state="all")
         pytest_assert(
-            wait_until(wait + 120, 10, 0, sonic_host.check_bgp_session_state_all_asics, bgp_neighbors),
+            wait_until(wait + 300, 10, 0, sonic_host.check_bgp_session_state_all_asics, bgp_neighbors),
             "Not all bgp sessions are established after config reload",
```

Comment:
Can you change the wait time when calling this function instead of changing the default wait time?